.github/workflows/gpu.yml

```diff
@@ -26,9 +26,14 @@ jobs:
       run: |
         ref_type=${{ github.ref_type }}
         branch=main
+        pr_flag="false"
         if [[ $ref_type == "tag"* ]]
         then
           git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/release*:refs/remotes/origin/release*
           branch=$(git branch -r --contains ${{ github.ref_name }} --list '*release*' --format "%(refname:short)" | sed -e 's/^origin\///')
         fi
-        cd ${{ github.workspace }}; tox -e py38-gpu -- $branch
+        if [[ "${{ github.event_name }}" == "pull_request" ]]
+        then
+          pr_flag="true"
+        fi
+        TOX_BRANCH=$branch TOX_PR_FLAG=$pr_flag cd ${{ github.workspace }}; tox -e py38-gpu
```

Comment:
`MERLIN_BRANCH` might be a name that matches the intent more narrowly? since it's the branch that we're going to use for the merlin packages install.